### PR TITLE
Add devextreme-aspnet-data w/ npm auto-update

### PR DESCRIPTION
**Pull request by OlegKipchatov**

### CODE REVIEW

**Comment by OlegKipchatov** [comment]
Hello @MattIPv4,
The check says that the latest version of the package does not contain the file specified in 'filename', but I definitely see it in the package. Would you please describe the issue in greater detail?

File: packages/d/devextreme-aspnet-data.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "devextreme-aspnet-data",
+  "filename": "js/dx.aspnet.data.js",
```
Comment (MattIPv4) [inline packages/d/devextreme-aspnet-data.json:3]:
```suggestion
  "filename": "dx.aspnet.data.js",
```
You're using basePath `js`, so the files are pulled up from that subdirectory into the root :)

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/d/devextreme-aspnet-data.json (hunk 1 above)
Comment (OlegKipchatov) [reply packages/d/devextreme-aspnet-data.json:3]: Thank you very much :)

**Review by MattIPv4 — APPROVED** [review]
Lgtm

**Pull request merged**